frontend/src/hooks/api/actions/useActionsApi/useActionsApi.ts

```diff
@@ -15,15 +13,18 @@ export type ActionSetPayload = Omit<
     actions: ActionPayload[];
 };
 
-export const useActionsApi = () => {
+export const useActionsApi = (project?: string) => {
     const { loading, makeRequest, createRequest, errors } = useAPI({
         propagateErrors: true,
     });
+    const endpoint = project
+        ? `api/admin/projects/${project}/actions`
+        : 'api/admin/actions';
```

Comment:
Do we want this? 🤔

Reply:
Not sure, I was hesitant to remove this, I think it's not needed given that I removed the endpoints :D